tests/e2e/workloads/pgsql/test_pgsql.py

```diff
@@ -38,11 +43,25 @@ def test_sql_workload_simple(self, pgsql):
             replicas=3, clients=3, transactions=600
         )
 
+        # Start measuring time
+        start_time = datetime.now()
+
+        # Check worker node utilization (adm_top)
+        get_node_resource_utilization_from_adm_top(node_type='worker', print_table=True)
+
+        # Check worker node utilization (oc_describe)
```

Comment:
There is no need to get the resource utilization from both adm top and oc describe.
We can go with the just adm top since that is ocp recommendation.